azure_img_utils/cloud_partner.py

```diff
@@ -28,6 +27,9 @@
 from requests.exceptions import HTTPError
 
 INGESTION_API = 'https://graph.microsoft.com/rp/product-ingestion/'
```

Comment:
May be we should change the globals' names to the same format? 
Changing this one to `ingestion_api` is ok for me, but having one all caps and the rest lowercase seems strange.

Reply:
Ack, fixed all to be upper case and rebased.